core/templates/pages/story-editor-page/navbar/story-editor-navbar.directive.ts

```diff
@@ -75,32 +81,37 @@ angular.module('oppia').directive('storyEditorNavbar', [
             $scope.validationIssues = $scope.story.validate();
             var nodes = $scope.story.getStoryContents().getNodes();
             var explorationIds = [];
-            for (var i = 0; i < nodes.length; i++) {
-              if (
-                nodes[i].getExplorationId() !== null &&
-                nodes[i].getExplorationId() !== '') {
-                explorationIds.push(nodes[i].getExplorationId());
-              } else {
-                $scope.validationIssues.push(
-                  'Some chapters don\'t have exploration IDs provided.');
-              }
-            }
 
-            if (StoryEditorStateService.getExpIdsChanged()) {
-              StoryEditorStateService.resetExpIdsChanged();
-              EditableStoryBackendApiService.validateExplorations(
-                $scope.story.getId(), explorationIds
-              ).then(function(validationIssues) {
-                $scope.explorationValidationIssues = validationIssues;
-                $scope.validationIssues =
-                  $scope.validationIssues.concat(
-                    $scope.explorationValidationIssues);
-              });
-            } else {
-              $scope.validationIssues =
-                $scope.validationIssues.concat(
-                  $scope.explorationValidationIssues);
+            if (
+              StoryEditorStateService.getExpIdsChanged() ||
```

Comment:
getChangedExpIds()?

Reply:
I'm not getting the changed exp IDs here, but just checking whether any exp IDs were changed at all, it returns a boolean. Should I rename it to something else then?

Reply:
Yes -- how about areAnyExpIdsChanged()?

Reply:
Done